addons/website/static/src/snippets/s_chart/options.js

```diff
@@ -476,9 +477,18 @@ options.registry.InnerChart = options.Class.extend({
     },
     /**
      * @private
-     * @param {Event} ev
      */
-    _onMatrixInputFocusOut: function (ev) {
+    _onMatrixInputBlur: function (ev) {
+        if (!this.el.contains(ev.relatedTarget)) {
+            // We call the functions below only if the element that recovers the
+            // focus after this blur is not an element of the we-matrix.
```

Comment:
Actually `this.el` here (in the "if" condition) is `we-customizeblock-option` and not `we-matrix`.
So is there not a change to be made in the code or in the comment?

Reply:
While thinking about that, the we-matrix should not do anything when we blur it... As we do a `reloadGraph()` when there is an input event, we don't need this this `_onMatrixInputBlur` / `_onMatrixInputFocusOut` anymore. I remove it :+1: